### PR TITLE
s-hydrosolver: Add EFICAS_NOUVEAU_ROOT variable -> EFICAS_TOOLS_ROOT_DIR

### PR DESCRIPTION
**Pull request by estreby**
New Eficas variable needed by s-hydrosolver

### CODE REVIEW

File: scripts.d/s-hydrosolver [hunk 1]
```diff
@@ -49,6 +49,7 @@ function s-hydrosolver-env()
 
     set-var    HYDROSOLVER_ROOT_DIR $PREFIX
     set-var    HYDROSOLVER_DIR      $PREFIX
+    set-var    EFICAS_NOUVEAU_ROOT  $EFICAS_TOOLS_ROOT_DIR
```
Comment (TurboGit) [inline scripts.d/s-hydrosolver:52]: this sounds wrong as `EFICAS_NOUVEAU_ROOT` and `EFICAS_TOOLS_ROOT_DIR` are both defined in `s-eficas-tools`.

So I suppose the proper fix is to add `s-eficas-tools` as dependency in s-hydrosolver-common-depends. Can you check that?

Comment (estreby) [reply scripts.d/s-hydrosolver:52]: It's in the case where eficas is carried by ng mode.
So, we can't use s-eficas-tools varaibles

Comment (TurboGit) [reply scripts.d/s-hydrosolver:52]: I see, so this must be set in `s-salome-bin` then. We don't want to set an eficas variable here, if it comes from binary release then we must define it at the right place.

Comment (estreby) [reply scripts.d/s-hydrosolver]: Update, I hope it will be better !

**Review by TurboGit — APPROVED** [review]
Looks good to me as-is. Thanks.

**Pull request merged**